src/www/app/outline_server.ts

```diff
@@ -21,6 +21,9 @@ import {Server} from '../model/server';
 import {PersistentServer} from './persistent_server';
 
 export class OutlineServer implements PersistentServer {
+  private static readonly SUPPORTED_CIPHERS =
```

Comment:
Add a comment with a reference to https://shadowsocks.org/en/spec/AEAD-Ciphers.html